README.md

```diff
@@ -17,7 +17,7 @@ Pepr is on a mission to save Kubernetes from the tyranny of YAML, intimidating g
 
 ## Features
 
-- Zero-config K8s webhook mutations and validations
+- Zero-config K8s Mutating and Validating Webhook and Controller generation
```

Comment:
```suggestion
- Zero-config K8s Mutating and Validating Webhooks plus Controller generation
```

I think webhooks ought to be plural here. Stylistically, saying "A and B and C" feels "bumpy" to me since webhooks and controller generation are different topics